units/recalculate_change_points.go

```diff
@@ -99,7 +99,7 @@ func (j *recalculateChangePointsJob) Run(ctx context.Context) {
 		sort.Slice(series.TimeSeries, func(i, j int) bool {
 			return series.TimeSeries[i].Order < series.TimeSeries[j].Order
 		})
-		var float_series []float64
+		float_series := make([]float64, 0)
```

Comment:
why not make([]float64, len(series.TimeSeries), and then avoid append and just set indexes....

Reply:
oh yeah i missed the for loop on line 103, this makes more sense

Reply:
Sure, done